subprojects/emcee/gradle-plugin/src/main/kotlin/com/avito/emcee/EmceeTestTask.kt

```diff
@@ -29,41 +31,55 @@ public abstract class EmceeTestTask : DefaultTask() {
     @get:Input
     public abstract val deviceApis: ListProperty<Int>
 
-    @get:Internal
+    @get:Internal // todo do we want to invalidate test results on host change?
+    public abstract val baseUrl: Property<String>
+
+    @get:Internal // todo do we want to invalidate test results on timeout increase?
     public abstract val testTimeout: Property<Duration>
 
-    @get:InputFile
-    public abstract val apk: Property<RegularFile>
+    @get:InputDirectory
+    public abstract val apk: DirectoryProperty
+
+    @get:InputDirectory
+    public abstract val testApk: DirectoryProperty
+
+    @get:Input
+    public abstract val configTestMode: Property<Boolean>
 
-    @get:InputFile
-    public abstract val testApk: Property<RegularFile>
+    @get:OutputDirectory
```

Comment:
This will be used for caching but we used this prop only in testing.
I propose to mark property `Internal`

Reply:
Internal until we decided how to store report as output?

Reply:
Until it behaves as an output with Gradle output semantics